### ISSUE TITLE
Wrong activity timeline URL: "Could not find any posts:"

### ISSUE THREAD

**Issue by kahliloppenheimer** [weskerfoot/DeleteFB#149]
The URL that the tool attempted to load appears to be broken.

![image](https://user-images.githubusercontent.com/6416509/103816468-02797300-5033-11eb-9d4e-bbca4b5bae81.png)

**Comment by weskerfoot** [comment]
will look into it, thanks for reporting!

**Comment by weskerfoot** [comment]
I guess this is because the year by year feature is broken. I had an idea for how to fix it, but IIRC it's tricky

**Comment by wrapper** [comment]
So you can no longer delete by year? I feel a bit reluctant to use the tool without being able to narrow down the criteria first

**Comment by weskerfoot** [comment]
> So you can no longer delete by year? I feel a bit reluctant to use the tool without being able to narrow down the criteria first

That's right, the fix is going to be a bit tricky because the new facebook site doesn't give you an easy way to filter year by url like it did previously. I started working on a fix for it though.

**Comment by danielfcastro** [comment]
I had the same issue although I know and manually checked that there are posts in a specific year.

**Comment by weskerfoot** [comment]
Going to take another crack at fixing this

**Comment by AlexKalopsia** [comment]
Thanks for looking into this @weskerfoot . From what I can see it is still somewhat possible to find the posts (not from mobile url) by using `allactivity/?category_key=STATUSCLUSTER&year=YYYY`

The annoyance is that the "..." Button only shows up on hover and I am not sure if selenium can handle it?

**Comment by weskerfoot** [comment]
> Thanks for looking into this @weskerfoot . From what I can see it is still somewhat possible to find the posts (not from mobile url) by using `allactivity/?category_key=STATUSCLUSTER&year=YYYY`
> 
> The annoyance is that the "..." Button only shows up on hover and I am not sure if selenium can handle it?
> 
> The alternative would still be to go from mobile on `allactivity/?category_key=all`, and find the `<header>YYYY</header` element

It can handle it, it's just really tricky to do it, you would have to use raw javascript I think

**Comment by slickwarren** [comment]
any update on this, @weskerfoot ? 
A potential workaround would be to start from the first post on the page, instead of the most recent, by default (I'm assuming if I try this without using the year, it starts with the most recent posts, right?)

**Comment by weskerfoot** [comment]
> any update on this, @weskerfoot ?
> A potential workaround would be to start from the first post on the page, instead of the most recent, by default (I'm assuming if I try this without using the year, it starts with the most recent posts, right?)

I haven't tried it out in a while, I still have the branch I started though. I'll test it out again this week and see if there's an easy fix

**Comment by weskerfoot** [comment]
Still planning on taking a look at it

**Comment by bradleygrant** [comment]
Hey, this is still ongoing, have you had any chance to look back in on this? At one time I was able to pull up legacy pages by changing the UserAgent string. I'll see if that's an option again.

**Comment by weskerfoot** [comment]
That would be helpful. This kind of fell by the wayside but I can merge any
fixes.
